### ISSUE TITLE
Markdown Note source

### ISSUE THREAD

**Issue by uGeek** [TriliumNext/Trilium#3049]
### Describe feature

Would it be possible to add the Note Source Markdown option, to display all the code in Markdown? same as it does in html but another option below in Markdown syntax.

Thank you very much and congratulations for making Trilium the best service and most complete of notes

**Comment by zadam** [comment]
Hi, note source is to display underlying storage of the note and for text notes we use HTML.

You can use the export dialog to convert the note into markdown:

![image](https://user-images.githubusercontent.com/617641/183488615-659e2816-fc05-416b-ac96-23b15e352a1f.png)

**Comment by uGeek** [comment]
Yes, I know. I said it so I don't have to export a file and save it. Being able to copy directly without the need to export. Copy all or part of the note if necessary.

For example, I generate my articles or notes with Trilium and if I want to publish them on the blog, it is tedious to have to export it to a file. In the end it is not a problem for me, because I have made a pandoc script that converts from html to markdown, I have no problem, but it would be very practical if the option existed, since markdown is much more friendly than html. But the current option of displaying the source note in html I also like.

The current export methods in files generating the directories, subdirectories,.... and all your notes is fantastic.

I meant aside from the current export methods, preview the note with markdown syntax to copy directly from the browser.

**Comment by meichthys** [comment]
@uGeek 
Copying from Trilium to Blog probably wouldn't work well anyway since the Headings in Trilium are mangled slightly. See here: https://github.com/zadam/trilium/issues/1369#issuecomment-798830025

**Comment by uGeek** [comment]
it's fantastic!!!

thank you

**Comment by eliandoran** [comment]
Closing in favor of #5645 which involves simply copying to Markdown. I don't see much merit in view the markdown source as opposed to an export.
